sentry-actix/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2018"
 
 [dependencies]
 sentry-core = { version = "0.22.0", path = "../sentry-core", default-features = false }
```

Comment:
I need to add this for my project using actix-web v4 and this sentry-actix branch to compile for some reason:

```suggested
sentry-core = { version = "0.22.0", path = "../sentry-core", default-features = false, features=["client"] }
```

Reply:
I also had to do this in my local fork

Reply:
Hmm, not sure about this as it seems to be working in my project and wasn't needed in previous versions.

Reply:
Ah, just managed to reproduce this. Honestly not sure why it wasn't happening before. It does appear it is now necessary though so I've updated the PR.